test/e2e/mcd_test.go

```diff
@@ -269,10 +269,14 @@ func TestExtensions(t *testing.T) {
 	assert.Equal(t, infraNode.Annotations[constants.CurrentMachineConfigAnnotationKey], renderedConfig)
 	assert.Equal(t, infraNode.Annotations[constants.MachineConfigDaemonStateAnnotationKey], constants.MachineConfigDaemonStateDone)
 
-	installedExtesnions := execCmdOnNode(t, cs, infraNode, "chroot", "/rootfs", "rpm", "-qa", "usbguard")
-	if !strings.Contains(installedExtesnions, "usbguard") {
-		t.Fatalf("Node %s doesn't have expected extensions", infraNode.Name)
+	installedExtesnions := execCmdOnNode(t, cs, infraNode, "chroot", "/rootfs", "rpm", "-q", "usbguard", "kernel-devel", "kernel-headers")
```

Comment:
Not new but tyop here: `installedExtensions`

Reply:
fixed it. variable has been also renamed to installedPackages which I missed to update here. Somehow, my VS code didn't catch the error(which I need to fix).